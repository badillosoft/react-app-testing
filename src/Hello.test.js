import React from "react";

import renderer from "react-test-renderer";

import Hello from "./Hello";

function HelloTest() {
    return <Hello name="Paco" />
}

test("Prueba de <Hello>", () => {
    const componentHello = renderer.create(
        <HelloTest />
    );

    const treeHello = componentHello.toJSON();

    console.log(treeHello);

    //expect(treeHello).toMatchSnapshot();
});