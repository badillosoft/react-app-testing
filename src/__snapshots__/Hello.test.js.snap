// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba de <Hello> 1`] = `
<h1>
  Hola mundo
</h1>
`;
